gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspAATask.kt

```diff
@@ -317,8 +317,11 @@ abstract class KspAATask @Inject constructor(
                         val konanTargetName = kotlinCompilation.target.konanTarget.name
                         cfg.konanTargetName.value(konanTargetName)
                         cfg.konanHome.value((kotlinCompileProvider.get() as KotlinNativeCompile).konanHome)
+
+                        val isKlibCrossCompilationEnabled =
+                            project.findProperty("kotlin.native.enableKlibsCrossCompilation") == "true"
                         kspAATask.onlyIf {
-                            HostManager().enabled.any {
+                            isKlibCrossCompilationEnabled || HostManager().enabled.any {
```

Comment:
I wonder if we should replace the entire logic in the `onlyIf` by `kotlinCompilation.target.publishable`. 

CC'ing @Tapchicoma,